modules/blocks.php

```diff
@@ -190,3 +190,72 @@ function jetpack_slideshow_block_load_assets( $attr, $content ) {
 	Jetpack_Gutenberg::load_assets_as_required( 'slideshow', $dependencies );
 	return $content;
 }
+
+/**
+ * Mailchimp Block.
+ */
+jetpack_register_block(
```

Comment:
Since we rely on `Jetpack_Options::get_option( 'id' )` in the block, we'll need to add a check for `Jetpack::is_active()` or wpcom before we register the block.

Reply:
Done in https://github.com/Automattic/jetpack/pull/11092/commits/d85588112d8eca5e8ca70eeed84dad86e8ad0bcb